src/Microsoft.Health.Fhir.Core/Messages/BulkImport/ImportBundleResponse.cs

```diff
@@ -0,0 +1,20 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Core.Messages.Import
+{
+    public class ImportBundleResponse
+    {
+        public ImportBundleResponse(int count)
+        {
+            LoadedResources = count;
+        }
+
+        /// <summary>
+        /// Number of loaded resources
+        /// </summary>
+        public int LoadedResources { get; }
```

Comment:
We should discuss about the expected outcomes of these bundles.

Reply:
Not sure what you mean. If all resources are valid, all resources are loaded, either updated, or created. Therefore outcome is simpler than for generic bundle.